server/views/add-articles.hbs

```diff
@@ -9,4 +9,9 @@
     <br/>
     <button type="submit">Send</button>
 </form>
- 
\ No newline at end of file
+ 
+<hr>
+
+ <nav class="nav">
+  <a class="nav-link" href="#">Add an Article </a>
+</nav>
```

Comment:
What was the reason for adding this nav at the bottom?

I think we can just revert this file?